NYPLCardCreator/Flow/NameAndEmailViewController.swift

```diff
@@ -140,13 +138,20 @@ final class NameAndEmailViewController: FormTableViewController {
   
   @objc override func didSelectNext() {
     self.view.endEditing(false)
-    
+
+    guard let firstName = firstNameCell.textField.text,
+      let middleInitial = middleInitialCell.textField.text,
+      let lastName = lastNameCell.textField.text else {
+        return
```

Comment:
I guess we don't need an alert or error or field highlighting because in theory this shouldn't be able to happen?

Reply:
Oh I see. This is just fetching the string value and empty string valid.
But if for some reason we can't fetch text still, I think we should log some sort of error so that we can be alerted to this unexpected case.

Reply:
I was thinking the same thing, but the card creator framework doesn't have that infrastructure, and I didn't want to complicate the logic too much for this very edgy case. I mainly wanted to improve readability and avoid the force unwraps. 

Also I'm hesitant / resisting my impulse to do more extensive refactors in this code base (for instance the URLSession logic could be improved too) because we'll remove it from SimplyE anyway once we have the web-based card creation.